tools/helios-build/src/main.rs

```diff
@@ -575,28 +575,31 @@ fn regen_illumos_sh<P: AsRef<Path>>(
     env += "export GNUC_ROOT=/opt/gcc-10\n";
     env += "export PRIMARY_CC=gcc10,$GNUC_ROOT/bin/gcc,gnu\n";
     env += "export PRIMARY_CCC=gcc10,$GNUC_ROOT/bin/g++,gnu\n";
+    env += "export SHADOW_CCS=\n";
+    env += "export SHADOW_CCCS=\n";
     match bt {
         BuildType::Quick | BuildType::QuickDebug => {
             /*
              * Skip the shadow compiler and smatch for quick builds:
              */
-            env += "export SHADOW_CCS=\n";
-            env += "export SHADOW_CCCS=\n";
         }
         BuildType::Full | BuildType::Release => {
             /*
-             * Enable the shadow compiler for full builds:
+             * Enable the shadow compiler(s) for full builds:
              */
-            env += "export SHADOW_CCS=gcc7,/opt/gcc-7/bin/gcc,gnu\n";
-            env += "export SHADOW_CCCS=gcc7,/opt/gcc-7/bin/g++,gnu\n";
+            for v in vec![14].iter() {
```

Comment:
Something like this perhaps?

```suggestion
            const GCC_VERSIONS: &[u32] = &[14];
            
            for v in GCC_VERSIONS {
```

Reply:
I like that, thanks.